project/webapp/templates/thread.html

```diff
@@ -90,4 +90,5 @@
     //
     // if (cw.ws.readyState == WebSocket.OPEN) cw.ws.onopen();
 </script>
+
```

Comment:
Go ahead and delete this line too. It should basically look like this file was untouched in the PR, since we no longer have any changes here.